Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM webdevops/php-nginx:8.2
+
+WORKDIR /var/www/html
+
+ENV WEB_DOCUMENT_ROOT /var/www/html/public
+
+COPY . /var/www/html
+
+RUN composer install --no-interaction
+
+RUN chmod +x entryfile.sh
+
+# Timeout for the post update script in minutes.
+LABEL com.centurylinklabs.watchtower.lifecycle.post-update-timeout="1440"
+# Watchtower will run this script after restarting the updated container.
+LABEL com.centurylinklabs.watchtower.lifecycle.post-update="/var/www/html/entryfile.sh"
+
+RUN chmod 777 -R ./storage/
```

Comment:
can be 755 i think

Reply:
yes, but the owner has to be application

RUN chown -R application:application /var/www/html/storage